aea/helpers/async_utils.py

```diff
@@ -44,15 +44,6 @@
 )
 
 
-try:
-    from asyncio import create_task  # pylint: disable=ungrouped-imports,unused-import
```

Comment:
Not sure why these imports where there in the first place. They don't get used...